### ISSUE TITLE
Villagers in modded structures spawn with the name "moog" and nothing else

### ISSUE THREAD

**Issue by BoyManDuck** [Luke100000/minecraft-comes-alive#545]
Villagers that spawn in any structures from mods always spawn with the name "moog" but not in vanilla or modded villages so its only with stand alone structures that spawn villagers.

![2023-01-28_20 24 17](https://user-images.githubusercontent.com/122084221/215301096-98c4af52-7e49-4725-9612-779f694f76c5.png)
![2023-01-28_20 22 08](https://user-images.githubusercontent.com/122084221/215301099-85da19e1-9dd6-483d-9e6d-643b796ecaf0.png)
 
Forge-43.2.4

minecraft-comes-alive-7.4.0+1.19.2-universal.jar

**Comment by Luke100000** [comment]
Do you have https://www.curseforge.com/minecraft/mc-mods/moogs-voyager-structures installed?

**Comment by BoyManDuck** [comment]
Yes, is that the mod that causes the glitch?

**Comment by Luke100000** [comment]
Yes, it spawns villagers with that name. I will need to figure out why the mod does that and then I might be able to fix that
